charts/knuto/README.md

```diff
@@ -16,5 +16,14 @@ Current chart version is `0.7.0`
 | kafkauser_source_namespaces.dev.deletion_enabled | bool | `true` |  |
 | kafkauser_source_namespaces.latest.deletion_enabled | bool | `false` |  |
 | kafkauser_source_namespaces.production.deletion_enabled | bool | `false` |  |
-| secret_type_to_bootstrap_server | object | `{"scram-sha-512":"production-kafka-bootstrap.kafka.svc.cluster.local:9092"}` | MApping of secret type to the DNS name an port of the Kafka service. Used to construct kafka-client.properties in Secrets placed in the namespaces configured in kafkauser_source_namespaces |
+| kafkauser_source_namespaces.dev.write_allowed_non_namespaced_topics | list of strings | [] |  |
```

Comment:
Please add description for the parameters. Not clear to me what "...non_namespaced_topics" mean.

And is it more clear to write "permission" or "access" instead of "allowed"? Or something like "write_permission_for_topics"?

Reply:
non_namepaced_topics refers to topics which has not been prefixed with the namespace which the KNUTO instance is operating on. Please read https://github.com/niradynamics/kafka-namespaced-user-topic-operator#conventions for more details. I have added descriptions which should make it a bit clearer. 

The thing is that KNUTO only forwards kafkausers and the created secrets, so I think allowed is more correct in this context than permission and access which is more for authorization. 

 Since by default are all topics for the same namespace allowed to write to makes an option like write_permission_for_topics not so precis, it makes me think that is is only the topics which is in that list that is has permissions to and that is not true.